packages/vite-plugin/package.json

```diff
@@ -15,9 +15,7 @@
   "author": "SEEK",
   "license": "MIT",
   "dependencies": {
-    "@vanilla-extract/integration": "^1.1.0"
-  },
-  "devDependencies": {
+    "@vanilla-extract/integration": "^1.1.0",
```

Comment:
Not sure why this was changed? Can you please revert this?

Reply:
https://github.com/seek-oss/vanilla-extract/blob/7c396ea58f5af60e65176875ae9c93764fe77fd2/packages/vite-plugin/src/index.ts#L2

Since we need to import the `normalizePath` method and use it, we need to adjust the `vite` dependencies to `dependencies` or `peerDependencies`, otherwise an exception will occur at build.
![image](https://user-images.githubusercontent.com/7699524/129499953-c0283698-d598-4570-8ace-1b214e0c98c6.png)

Maybe I should have adjusted it to `peerDependencies` more appropriately.

Reply:
Right. Sorry I misread this change. I think `peerDependencies` is the right call here.

Reply:
Done, thank you for the review.

Reply:
Sorry but you'll need to leave the dev dependency there as well, otherwise it may not be accessible locally.

Reply:
Sorry, I don't quite understand what you mean, because it is a plugin for `vite`, there is no way to run it alone, it can only work with `vite`, so `vite`must exist.